_posts/2019-04-29-compatibility-table.md

```diff
@@ -28,35 +28,35 @@ Rails we have ran into a lot of these combinations.
     <tr>
       <td>6.1.0</td>
       <td>&gt;= 2.5.0</td>
-      <td></td>
+      <td>2.7.1</td>
       <td>&gt;= 1.8.11</td>
       <td>Alpha</td>
     </tr>
     <tr>
       <td>6.0.x</td>
-      <td>&gt;= 2.5.0</td>
-      <td></td>
+      <td>&gt;= 2.5.0<br/><a href="https://github.blog/2020-08-25-upgrading-github-to-ruby-2-7/" target="_blank">&lt; 3.0.0</a></td>
```

Comment:
@etagwerker is it already confirmed that rails 6 will support all ruby versions `< 3.0`?

Reply:
@lubc There won't be a Ruby 2.8 version because they are jumping from 2.7.x to 3.0.x. And this article explains how GitHub upgraded to Ruby 2.7.x -- The part that I'm not sure about: Not sure if GitHub is running Rails 6.1 or 6.0 in production.

Reply:
@etagwerker got it 👍 , they're running `6.0` as far as i know. re: https://github.blog/2019-09-09-running-github-on-rails-6-0/

Reply:
Just tried to create a new rails app using both 6.0.3 and 6.0.2, with 6.0.2 it works just fine, with 6.0.2 it works but with many warning (just started the server and entered the default index page and saw 8 warnings in the console.

Reply:
Maybe we can add an `*` saying that rails 6.0.3 is recommended for ruby 2.7.1?